api/managementpb/dbaas/components.proto

```diff
@@ -102,6 +102,39 @@ message ChangePXCComponentsRequest {
 
 message ChangePXCComponentsResponse {}
 
+// OperatorUpdateStatus represents status of an update for the operator.
+enum OperatorUpdateStatus {
+  OPERATOR_UPDATE_STATUS_INVALID = 0;
+  // An update of the operator is available for the installed version of PMM server.
+  UPDATE_AVAILABLE = 1;
+  // An update of the operator is not available for the installed version of PMM server.
+  UPDATE_NOT_AVAILABLE = 2;
+}
+
+message CheckForOperatorUpdateRequest {}
+
+// OperatorUpdateInformation indicates if operator update is available. It includes what version of operator we can update to.
+// If we are running latest operator for installed PMM version and more up-to-date operator is available, we also get PMM
+// version we can update to to get the most up-to-date operator.
+message OperatorUpdateInformation {
+  // The status indicates operator status availablility.
+  OperatorUpdateStatus status = 1;
+  // available_operator_version is not empty if there is an update available,
+  // for both compatible and incompatible one.
+  string available_operator_version = 2;
+}
```

Comment:
Feels like we will be able to reuse it later for DBs. Maybe it's better to make it somehow common?
```suggestion
message ComponentUpdateInformation {
  // The status indicates operator status availablility.
  ComponentUpdateStatus status = 1;
  // available_component_version is not empty if there is an update available.
  string available_component_version = 2;
}
```
Do we really need `ComponentUpdateStatus` if `available_component_version` is not empty only when update is available?

Reply:
I removed it.

Reply:
For the common thing: Since I removed status, I could place operator specific fields right into OperatorUpdateInformation. One way to generalize it would be to have map with key as operator type and value available version. what do you think @BupycHuk ?

Reply:
It can work, just to not forget to mention it in comments.

Reply:
But since we are going to add ability to notify later, maybe it's better to keep the message

Reply:
Let's solve this when we actually decide to show updates to incompatible versions are available.

Reply:
I changed the implementation to use maps, adding postresql operator will result in a little less pain in this case.

Reply:
the problem with this approach that we won't be able to change already made API.

Reply:
Why not? As long as we change FE and BE in the same release, it should be fine, shouldn't it?

Reply:
I added the one field message, so we are backwards compatible.